docs/quick-start.md

```diff
@@ -36,6 +36,18 @@ The second command will download and run a prebuilt AsterixDB docker container f
 After it finishes, you should see the messages as shown in the following screenshot:
 ![docker][docker]
 
+After you have installed the docker container for the first time, you may restart it later using the following command:
```

Comment:
It's good to have these instructions, but we need to make it very clear when they are needed.  How about the following:

"In case the docker container is shut down accidentally, you can restart it using the following command:"